app/services/force/soql/preference_service.rb

```diff
@@ -7,15 +7,44 @@ class PreferenceService < Force::Base
       FIELD_NAME = :preferences
       FIELDS = load_fields(FIELD_NAME).freeze
 
-      def application_preferences(application_id)
+      def application_preferences_for_application(application_id)
```

Comment:
what is an `application_preference_for_application`?
I'm also not super clear on what an `application_preference_for_listing` is

Reply:
Preferences are a bit complex in the domain model but from what I gathered, the `Application_Preference` object allows us to see preferences and most importantly their order and rank -- this is the object needed for the lease ups apps table since we need the order and rank. `application_preference_for_listing` was essentially `lease_up_service#lease_up_listing_applications` but the method name was not accurate because we weren't getting applications, we were getting `application_preferences` that were matched to the listing and then constructing the data so that we also get application fields from its related application. 

So to make it more RESTful we got rid of `lease_up_service` since it was not an actual resource, and moved `#lease_up_listing_applications` to `preference_service` and renamed it as `application_preference_for_listing`. And then renamed `application_preferences` to `application_preferences_for_application` (because of the `WHERE` clause`.